### DISCUSSION TITLE
Auto generation of MEPLHAA from Tasmota?

### DISCUSSION THREAD

**Discussion by cavanaug** [RavenSystem/esp-homekit-devices#1960] [MEPLHAA Scripts]
Ive been looking at many of the Tasmota supported devices via Base:18 and I think many could be auto translated into HAA scripts.

I think it could be pretty useful to have a broader collection of supported devices for HAA.    People that are Apple users, really really need to look at HAA.    But we need a better way to accelerate the switch to HAA and I think some type of translation would add many many devices.

I am super happy with the migration of all of my devices and in particular I was able to eliminate 2 entire docker containers and sw from my setup.   Basically move to HAA and you can eliminate an mqtt server and a homebridge server, huge win.   The best part, is no part...

**Comment by PhaseAngle** [comment]
Not quite what you are after but it will help
https://github.com/RavenSystem/esp-homekit-devices/discussions/1634

**Comment by PhaseAngle** [comment]
The code is a bit out of date 4 years hopefully someone might update it

https://github.com/glumb/haa-configurator
